doc/productivity.md

```diff
@@ -46,38 +46,33 @@ Note: base_dir is required for ccache to share cached compiles of the same file
 
 You _must not_ set base_dir to "/", or anywhere that contains system headers (according to the ccache docs).
 
-### Disable features with `./configure`
+### Disable features when generating the buildsystem
```

Comment:
build system is usually separated in existing code

Reply:
Thanks! Updated.